src/backend/marsha/core/serializers/live_session.py

```diff
@@ -195,6 +197,11 @@ def update(self, instance, validated_data):
                     }
                 )
 
+        if validated_data.get("is_registered") is True:
+            validated_data["registered_at"] = timezone.now()
+        elif validated_data.get("is_registered") is False:
+            validated_data["registered_at"] = None
+
```

Comment:
use of ternary ? 
```suggestion
      validated_data.get("registered_at") = timezone.now() if validated_data.get("is_registered",None) else None
```

Reply:
+ commit message

When a livesession is updated and is_registered is to True, we have to
**senf** the registration email. : senf  => send

Reply:
I will test but I don't want any modification if the `is_registered` field is not in the request.